coops-ui/src/components/EntityInfo.vue

```diff
@@ -108,11 +119,11 @@ export default class EntityInfo extends Vue {
 // see https://vue-loader-v14.vuejs.org/en/features/postcss.html
 @import '@/assets/styles/theme.scss';
 
-.entity-info {
+#entity-info {
   background: $BCgovInputBG;
 }
 
-// .entity-info.staff {
+// #entity-info.staff {
 //   background-image: url("data:image/svg+xml;utf8,<svg xmlns='http://www.w3.org/2000/svg' width='105' height='100'><text x='0' y='105' font-size='30' transform='rotate(-45 10,40)' opacity='0.1'>STAFF</text></svg>");
 //   background-repeat: repeat-x;
 // }
```

Comment:
Is this for a future feature as well?

Reply:
Sorta. If you uncomment this then you get a nice effect when the user is staff. I did this when I was developing staff roles.

On other projects we always had an indication when a staff was logged in and doing something on behalf of a user, so while it's not currently a requirement for us, it may one day become one, and this is my contribution to it :)

Reply:
Okie doke, sounds good!